src/pugi/pugixml.cpp

```diff
@@ -19,7 +19,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
-#include <assert.h>
```

Comment:
You should open a PR on pugixml and not change it here.

Reply:
Sorry. Actually there is one already: https://github.com/zeux/pugixml/pull/350/